src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneIndexWriter.cs

```diff
@@ -151,13 +151,16 @@ private void CreateIndexWriter(IState state)
         {
             _indexWriter = new TimeTrackingIndexWriter(_directory, _analyzer, _indexDeletionPolicy, _maxFieldLength, state);
             _indexWriter.UseCompoundFile = false;
-            _indexWriter.SetMergePolicy(new LogByteSizeMergePolicy(_indexWriter)
+            var mergePolicy = new LogByteSizeMergePolicy(_indexWriter)
             {
                 MaxMergeMB = _index.Configuration.MaximumSizePerSegment.GetValue(SizeUnit.Megabytes),
                 MergeFactor = _index.Configuration.MergeFactor,
                 LargeSegmentSizeMB = _index.Configuration.LargeSegmentSizeToMerge.GetValue(SizeUnit.Megabytes),
                 NumberOfLargeSegmentsToMergeInSingleBatch = _index.Configuration.NumberOfLargeSegmentsToMergeInSingleBatch
-            });
+            };
+            
+            mergePolicy.SetUseCompoundFile(_index.Configuration.LuceneUseCompoundFileInMerging);
```

Comment:
The only reason to use this is for the buffering, no?

Reply:
Answer in https://github.com/ravendb/ravendb/pull/14755/files#r960288572